docs/releases/1.11.txt

```diff
@@ -483,6 +483,10 @@ Miscellaneous
   ``migrate`` command. Use the new :djadmin:`remove_stale_contenttypes` command
   instead.
 
+* ETags are now parsed according to the updated RFC 7232 Conditional Requests
+  specification. The ETag syntax there is slightly different from that of the
+  older RFC 2616 specification which was used in previous versions.
```

Comment:
In my opinion an example would be overkill. My guess is that this change won't affect anyone, but if it does it will be because they're supplying their own, unusually-formatted ETags, and in that case they should really be consulting the RFCs directly. Let me know if you disagree.